ui/raidboss/data/06-ew/raid/p5s.ts

```diff
@@ -109,13 +115,41 @@ const triggerSet: TriggerSet<Data> = {
       durationSeconds: 5,
       response: Responses.getFrontThenBack(),
     },
+    {
+      id: 'P5S Raging Tail Move',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '7A0C', source: 'Proto-Carbuncle', capture: false }),
+      infoText: (_data, _matches, output) => output.moveBehind!(),
+      outputStrings: {
+        moveBehind: {
+          en: 'Move (Behind)',
+        },
+      },
+    },
     {
       id: 'P5S Claw to Tail',
       type: 'StartsUsing',
       netRegex: NetRegexes.startsUsing({ id: '770E', source: 'Proto-Carbuncle', capture: false }),
       durationSeconds: 5,
       response: Responses.getBackThenFront(),
     },
+    {
+      id: 'P5S Raging Claw Move',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '770F', source: 'Proto-Carbuncle', capture: false }),
+      condition: (data) => {
+        return ++data.clawCount === 6;
```

Comment:
For the sake of readability, can you put the increment and the equality on separate lines?

Reply:
Done.